courses/free-courses-ta.md

```diff
@@ -94,6 +103,13 @@ CSE Tamila by Eezytutorials
 ### MongoDB
 
 * [MongoDB Course in Tamil](https://www.youtube.com/playlist?list=PL7BQ4lqtgECRiWoThupyKXRQoDuEV2zy5) - JVL Code
+* [MongoDB Tamil Tutorial for Beginners](https://www.youtube.com/playlist?list=PLfD4W8QfMd5DhXKriTHyHjNzNSe_1I7g1) - MaanavaN Learn Code
+
+
+### Next JS
+
+* [Full stack Next JS Project Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-GmZBy4-tPhwwL8AEjW2t8G) - Tamil Coding Wizard
+* [Next JS Beginner series Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ) - Tamil Coding Wizard
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
Can you please explain me which titles are you referring to?

Reply:
```suggestion
* [[Next JS Tamil \| Next JS beginner Series Tamil \| Next.js Framework Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ)](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ) - Tamil Coding Wizard
```